openedx/core/djangolib/markup.py

```diff
@@ -55,3 +56,20 @@ def strip_all_tags_but_br(string_to_strip):
     string_to_strip = bleach.clean(string_to_strip, tags=['br'], strip=True)
 
     return HTML(string_to_strip)
+
+
+def clean_dangers_html(html):
```

Comment:
This function name isn't clear. Can we change it to `clean_dangerous_html`?

Reply:
@tuchfarber updated